contracts/sources/obligation.move

```diff
@@ -167,6 +184,45 @@ module suilend::obligation {
         }
     }
 
+    public(friend) fun set_emode<P>(
+        obligation: &mut Obligation<P>,
+        deposit_reserve: &Reserve<P>,
```

Comment:
similarly to my comment in lending market, not sure why we need to know the deposit and borrow reserve

Reply:
Same response as above:

> When we call deposit for an emode obligation we need to be able to know what the corresponding borrow reserve is in order to get the respective emode ltv